edsdme/components/PartnerCards.js

```diff
@@ -198,7 +198,12 @@ export default class PartnerCards extends LitElement {
     const { search } = location || window.location;
     this.urlSearchParams = new URLSearchParams(search);
 
-    if (this.urlSearchParams.has('filters', 'yes')) {
+    const term = this.urlSearchParams.get('term');
+    if (term) {
+      this.searchTerm = term;
+    }
+
```

Comment:
There are some side effects of this changes:
1) if user chooose 'clear all' on page, term would also be deleted from input but on reload term will be back
2) if user update term and reload page, old term that is in url will be used insted new one

so we should keep updated this new query param in all actions (clear all, on search term set )

Reply:
@SonjaPopovic update in https://github.com/adobecom/dme-partners/pull/93/commits/14b1f8761b9131829966184c8b812339435519b3

Reply:
Thanks. There is also handleResetActions() in this file, which is triggered when user  clicks 'clear all' , i guess url param should be deleted there also,since search term will be deleted